example/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
is it the same config as `../.prettierrc.js`?

Reply:
yep, we have 2 separate apps here, seems 2nd requires extra config